### PR TITLE
doc(1.8.0): add replica auto-balance to best-practice

### PR DESCRIPTION
**Pull request by c3y1huang**
#### Which issue(s) this PR fixes:

Issue `None`

#### What this PR does / why we need it:

Recommend setting the replica auto-balance setting option to least-effort as best practice for the production environment.

#### Additional documentation or context

https://suse.slack.com/archives/C02CLV7D2KH/p1726814707259759

### CODE REVIEW

File: content/docs/1.8.0/best-practices.md [hunk 1]
```diff
@@ -217,3 +217,13 @@ This setting should be set to `false` in production environment to ensure the be
 > Recommend: `false`
 
 This setting should be set to `false` in production environment to ensure every volume have the best availability when created. Because with the setting set to `true`, the volume creation won't error out even there is only enough room to schedule one replica. So there is a risk that the cluster is running out of the spaces but the user won't be made aware immediately.
+
+### Replica Auto-Balance
+
+> Recommend: `least-effort`
+
+For production environments, we recommend to set Replica Auto-Balance to `least-effort`. This setting ensures that at least one replica is placed on a different node and in each zone, providing extra high availability (HA).
```
Comment (jhkrug) [inline content/docs/1.8.0/best-practices.md:225]:
```suggestion
For production environments, we recommend setting Replica Auto-Balance to `least-effort`. This setting ensures that at least one replica is placed on a different node in each zone, providing extra high availability (HA).
```

**Review by jhkrug — CHANGES_REQUESTED** [review]
Small suggestion

**Review by jhkrug — APPROVED** [review]
ok, lgtm

**Review by innobead — APPROVED** [review]
LGTM

**Comment by innobead** [comment]
We can also add this to 1.7 and 1.6.

**Review by innobead — APPROVED** [review]

**Pull request merged**